### PR TITLE
fix delete and add user

### PR DESCRIPTION
**Pull request by MrAdupo**
This Pull request fixes delete and adding user.

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -1,64 +1,61 @@
...
     const id = +req.params.id
-    console.log(mm)
-        const data2 = db.findIndex(a => a.id === id)
-        console.log("before update" , db[data2])
-        db[data2] = mm
-        res.send (db[data2])
-        
-         })
+    const data2 = db.findIndex(a => a.id === id)
+    console.log("before update", db[data2])
```
Comment (multy188) [inline server.js]: remove console.log

Comment (MrAdupo) [reply server.js]: done

**Review by MrAdupo — COMMENTED** [review]
i had to commit this on fix-update branch but mistakenly did it on fix-delete branch. how do i resolve this issue?

**Review by multy188 — APPROVED** [review]

**Comment by MrAdupo** [comment]
i have merged successfully

**Pull request merged**